notes/expedited-screening.vbs

```diff
@@ -58,20 +59,26 @@ changelog_display
 
 'THE SCRIPT----------------------------------------------------------------------------------------------------
 EMConnect ""    'Connecting to BlueZone
+Call check_for_MAXIS(FALSE) 'checking for an active MAXIS session
 call MAXIS_case_number_finder(MAXIS_case_number) 'It will search for a case number.
 ' application_date = date & ""
 If MAXIS_case_number <> "" Then
-    Call navigate_to_MAXIS_screen("STAT", "PROG")
-    EMReadScreen snap_pend_check, 4, 10, 74
-    If snap_pend_check = "PEND" Then
-        EMReadScreen snap_app_date, 8, 10, 33
-        application_date = replace(snap_app_date, " ", "/")
-    End If
-    transmit
-    EMReadScreen check_for_hcre, 4, 2, 50
-    If check_for_hcre = "HCRE" Then
-        PF10
-    End If
+    Call navigate_to_MAXIS_screen_review_PRIV("STAT", "PROG", is_this_priv)
```

Comment:
Even though we allow this to continue, we shouldn't do this until after the dialog. We have had issues where the wrong case number is pulled and the worker should have a chance to check the case number before we just don't even try to case note.

Reply:
Moved to STAT/DISQ portion of script.